src/SprykerEco/Shared/AkeneoPim/AkeneoPimConstants.php

```diff
@@ -7,11 +7,14 @@
 
 namespace SprykerEco\Shared\AkeneoPim;
 
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
 interface AkeneoPimConstants
 {
-    public const HOST = 'AKENEOPIM:HOST';
-    public const USERNAME = 'AKENEOPIM:USERNAME';
-    public const PASSWORD = 'AKENEOPIM:PASSWORD';
-    public const CLIENT_ID = 'AKENEOPIM:CLIENT_ID';
-    public const CLIENT_SECRET = 'AKENEOPIM:CLIENT_SECRET';
+    public const HOST = 'AKENEO_PIM:HOST';
```

Comment:
These changes trigger a major. Are you sure this is in the scope for the current release?

Reply:
it wasn't critical for the problem i faced with. OK i'll roll it back

Reply:
This kind of BC breaking technical depth please note down and we fix it in the next major.